rllib/algorithms/algorithm_config.py

```diff
@@ -663,10 +670,12 @@ def update_from_dict(
         # Namely, we want to re-instantiate the exploration config this config had
         # inside `self.experimental()` before potentially overwriting it in the
         # following.
-        if "_enable_new_api_stack" in config_dict:
-            self.experimental(
-                _enable_new_api_stack=config_dict["_enable_new_api_stack"]
-            )
+        enable_rl_module_and_learner = config_dict.get(
```

Comment:
Why is this? Imo users using still this "old" form must switch anyways in near future - so they could do it now. It's a small change.

Reply:
It's mostly for ourselves :| 
Some scripts of ours work for both API stacks and we don't want to make the `exploration_config` dependent on this. I'm fully with you: This is currently very hackish and should be cleaned up. I'll create another PR to see, what the impact would be for our code base (and tests), if we get rid of this hack.